.gitignore

```diff
@@ -21,3 +21,6 @@ testem.log
 .node_modules.ember-try/
 bower.json.ember-try
 package.json.ember-try
+
+# intellij
+.idea/
```

Comment:
I guess I'm the first dev to update this repo in intellij

Reply:
I opted for globally ignoring https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer since I figured I'd be the only one accessing and working with Sublime Workspaces across Addepar.

Reply:
Ooh this is a good thing to know about